.travis.yml

```diff
@@ -149,16 +156,16 @@ script:
       travis_run_in_fold "make.install" sudo env PATH=$PATH make install
       awesome --version
     fi
-  - |
-    if [ "$TRAVIS_TEST_RESULT" = 0 ]; then
-      do_codecov samples
-    fi
   # Run check-unit{,-coverage} and check-integration.
   - |
     set -ev
     if [ "$TRAVIS_TEST_RESULT" = 0 ]; then
       if [ "$DO_COVERAGE" != "0" ]; then
         travis_fold_start "DO_COVERAGE"
+
+        (mkdir -p tests/examples/build && cd tests/examples/build && cmake $CMAKE_ARGS -DUSE_LCOV=1 "..")
+        do_codecov samples tests/examples/build
```

Comment:
Maybe there is a problem with relative paths (in `luacov.stats.out`, which gets copied).
What about `cd`ing here instead?!